shelley/chain-and-ledger/formal-spec/multi-sig.tex

```diff
@@ -377,17 +389,19 @@ \section{Types}
 
   \begin{equation*}
     \begin{array}{r@{~\in~}lr}
-      \fun{txins} & \Tx \to \powerset{(\TxIn \times \Script^{?})} & \text{transaction inputs}
+      \fun{txinsVKey} & \Tx \to \powerset{\TxIn} &\text{VKey locked transaction inputs} \\
+      \fun{txinsScript} & \Tx \to \powerset{(\TxIn \times \Script)} &\text{Script locked transaction inputs} \\
+      \fun{txwitsVKey} & \Tx \to (\VKey \mapsto \Sig) & \text{VKey witnesses} \\
+      \fun{txwitsScripts} & \Tx \to \powerset{\Script} & \text{script witnesses}
```

Comment:
This was a bit confusing to me at first, since it isn't really an accessor. Done this way, we have some of the "witnesses" outside the txbody and some inside it. I think that's fine, I'm just thinking of how best to make the presentation...